backend/lib/deal-observer.js

```diff
@@ -79,7 +79,9 @@ export async function storeActiveDeals (activeDeals, pgPool) {
           term_start_epoch,
           term_min,
           term_max,
-          sector_id
+          sector_id,
+          payload_retrievability_state,
```

Comment:
```suggestion
          payload_retrievability,
```

(everything stored in Postgres is a state)

If you want to annotate the type, maybe instead this?

```suggestion
          payload_retrievability_enum,
```

I suggest we wait for @bajtos with this one